### PR TITLE
Getting Started papercuts

### PR DESCRIPTION
**Pull request by cnunciato**
Walked through all of the new stuff and fixed up little things as I found them.

Fixes https://github.com/pulumi/marketing/issues/309.

Related PRs:
* https://github.com/pulumi/templates/pull/280
* https://github.com/pulumi/templates/pull/281
* https://github.com/pulumi/templates/pull/282

### CODE REVIEW

**Review by susanev — APPROVED** [review]

File: themes/default/layouts/shortcodes/install-java.html [hunk 1]
```diff
@@ -2,3 +2,15 @@
...
 </p>
+
+<div class="note note-info" role="alert">
+    <div class="note-heading">
+        <i class="fas fa-info-circle mr-1"></i>
+        <span>Note</span>
+    </div>
+    <div class="note-body">
+        <p>
+            Pulumi will need the <code>java</code>, <code>javac</code>, and <code>mvn</code> executables in order to build and run your Pulumi Java application. Ensure that these executables can be found on your path after installation.
```
Comment (nimbinatus) [inline themes/default/layouts/shortcodes/install-java.html:13]: Would it be worth linking to something like sdkman here?

**Review by nimbinatus — APPROVED** [review]
Just a couple questions, no blockers

File: themes/default/content/docs/get-started/kubernetes/modify-program.md [hunk 2]
```diff
@@ -368,16 +368,13 @@ import com.pulumi.kubernetes.core_v1.enums.ServiceSpecType;
...
 import java.util.Map;
 
-
 public class App {
     public static void main(String[] args) {
         Pulumi.run(ctx -> {
             var config = ctx.config();
-            var isMinikube = config.getBoolean("isMinikube").orElse(false); // Double check
-
+            var isMinikube = config.requireBoolean("isMinikube");
```
[the remark this answers is not in the document]
Comment (cnunciato) [reply themes/default/content/docs/get-started/kubernetes/modify-program.md:377]: Definitely worked for me this morning! I remember getting the Intellisense for all of the config methods also.

File: themes/default/layouts/shortcodes/install-java.html (hunk 1 above)
Comment (cnunciato) [reply themes/default/layouts/shortcodes/install-java.html:13]: Interesting idea, yeah. I was mainly just looking to add some details for Java (and for consistency with the other languages), but it raises a good point -- given how common and useful these kinds of version managers are now, it'd probably be nice if we had page, or a section for each page under Intro > Languages, specifically for making these kinds of recommendations. What do you think? I'll file an issue if it sounds like something useful.

File: themes/default/content/docs/get-started/kubernetes/modify-program.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (cnunciato) [reply themes/default/content/docs/get-started/kubernetes/modify-program.md:377]: image

**Pull request merged**